core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -289,7 +289,10 @@ public DockerCompose(File composeFile, String identifier) {
     public void start() {
         super.start();
 
-        this.followOutput(new Slf4jLogConsumer(logger()), OutputFrame.OutputType.STDERR);
+        Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(logger());
+        this.followOutput(logConsumer, OutputFrame.OutputType.STDOUT);
```

Comment:
If you use `followOutput` with just the logConsumer as a single param there's no need to specify each of the output types - the logConsumer will just receive all frame types.